src/transformers/models/camembert/configuration_camembert.py

```diff
@@ -36,12 +36,57 @@
 }
 
 
-class CamembertConfig(RobertaConfig):
+class CamembertConfig(PretrainedConfig):
     """
-    This class overrides [`RobertaConfig`]. Please check the superclass for the appropriate documentation alongside
-    usage examples. Instantiating a configuration with the defaults will yield a similar configuration to that of the
+    This is the configuration class to store the configuration of a [`CamembertConfig`] or a [`TFCamembertConfig`]. It is
+    used to instantiate a Camembert model according to the specified arguments, defining the model architecture.
+    Instantiating a configuration with the defaults will yield a similar configuration to that of the
     Camembert [camembert-base](https://huggingface.co/camembert-base) architecture.
 
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+
+    Args:
+        vocab_size (`int`, *optional*, defaults to 30522):
+            Vocabulary size of the BERT model. Defines the number of different tokens that can be represented by the
+            `inputs_ids` passed when calling [`BertModel`] or [`TFBertModel`].
```

Comment:
Should be
```suggestion
            Vocabulary size of the BERT model. Defines the number of different tokens that can be represented by the
            `inputs_ids` passed when calling [`CamembertModel`] or [`TFCamembertModel`].
```

Reply:
> Vocabulary size of the BERT model.

Should the BERT her be changed to Camembert? (Line 52)